.github/ISSUE_TEMPLATE/module_request.yaml

```diff
@@ -0,0 +1,42 @@
+name: Feature request - Wrap new GMT module
+description: Request PyGMT wrapper for a GMT module
+title: "Wrapper for <module-name>"
+labels: ["feature request"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please replace <module-name> in the issue title and the description with the name of the requested module and add the description of the module.
+  - type: textarea
+    id: which-module
+    attributes:
+      label: Description of the desired module
+      description: Please be as detailed as you can in your description. If possible, include an example of how you would like to use this feature (even better if it's a code example).
+      placeholder: Implement [`<module-name>`](https://docs.generic-mapping-tools.org/latest/<module-name>.html) which `<insert description of the GMT module>`.
+    validations:
+      required: true
+  - type: dropdown
+    id: help
+    attributes:
+      label: Are you willing to help implement and maintain this feature?
+      description: |
+        Every feature we add is code that we will have to maintain and keep updated. This takes a lot of effort.
+        If you are willing to be involved in the project and help maintain your feature, it will make it easier for us to accept it.
+      options:
+        - "No"
+        - "Yes"
+        - "Maybe"
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        **Checklist for wrapping GMT Module**
+
+        <!-- Please do not edit this section when submitting pull requests. This checklist will help track progress on wrapping the module. -->
+
+        - [ ] 'Initial feature implementation' Pull Request (PR)
+        - [ ] 'Add missing aliases to module' documentation PR
+        - [ ] 'Support additional functionality in module' PR (optional)
+        - [ ] 'Add gallery example for module' documentation PR
+        - [ ] 'Add tutorial for module' documentation PR (optional)
```

Comment:
Just want to confirm that we want to keep this section in the feature request template, versus using the project board as suggested https://github.com/GenericMappingTools/pygmt/pull/1687#discussion_r776421366. I don't have a strong opinion here, but if we are going to remove it in favor of the project board now's a good time to do so.

Reply:
I'm OK with using the project board. We can remove this checklist and add a link to the project board.

Reply:
Done in 44359182ddd35657a746b027d82f1fd7547ef35c.